rules/linux/credential_access_potential_linux_ssh_bruteforce_internal.toml

```diff
@@ -48,14 +48,16 @@ The rule identifies consecutive SSH login failures targeting a user account from
 
 """
 risk_score = 47
-rule_id = "1c27fa22-7727-4dd3-81c0-de6da5555feb"
+rule_id = "a8fb5071-c071-4ec0-9ecd-85b45244ddfc"
 severity = "medium"
 tags = ["Elastic", "Host", "Linux", "Threat Detection", "Credential Access"]
 type = "eql"
 query = '''
-sequence by host.id, source.ip, user.name with maxspan=10s
+sequence by host.id, source.ip, user.name with maxspan=5s
   [authentication where event.action  in ("ssh_login", "user_login") and
-   event.outcome == "failure" and source.ip != null and source.ip != "0.0.0.0" and source.ip != "::" ] with runs=10
+   event.outcome == "failure" and
+   cidrmatch(source.ip, "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16")
```

Comment:
Maybe you can use this one here: https://github.com/elastic/detection-rules/blob/f41c5288cc0ff45e6bc3d6707c053e853721cf50/rules/windows/defense_evasion_msxsl_network.toml#L31